{{cookiecutter.project_slug}}/clients/web/react/src/services/user/api.ts

```diff
@@ -14,12 +14,12 @@ const login = createCustomServiceCall(
   },
 )
 
-const requestPasswordResetCode = createCustomServiceCall(
+const requestPasswordReset= createCustomServiceCall(
   {
     inputShape: forgotPasswordShape,
   },
   async ({ client, input }) => {
-    await client.get(`api/password/reset/code/${input.email}/`)
+    await client.get(`api/password/reset/${input.email}/`)
```

Comment:
a change here but none to the backend?

Reply:
The backend is all set, we probably copied some of this code from an app that was using code only the mobile side needs this code!